cli/src/auth/okta.rs

```diff
@@ -248,36 +250,43 @@ pub async fn refresh_tokens(config: &Config) -> Result<OktaAuth, WKCliError> {
         .map_err(|_err| AuthError::OpenIDDiscoveryError)?;
 
     // Set up the config for the Okta OAuth2 process.
-    let client = CoreClient::from_provider_metadata(provider_metadata, okta_client_id, None)
-        .set_redirect_uri(
-            RedirectUrl::new("http://localhost:6758/login/callback".to_string())
-                .expect("Invalid redirect URL"),
-        );
+    let client = CoreClient::from_provider_metadata(provider_metadata, okta_client_id, None);
 
-    let token_response = client
+    let token_exchange_result = client
         .exchange_refresh_token(&RefreshToken::new(auth_config.refresh_token.clone()))
         .request_async(async_http_client)
-        .await
-        .map_err(|err| match err {
-            openidconnect::RequestTokenError::ServerResponse(error) => {
-                let error_description = error.to_string();
-                if error.error().to_string() == "invalid_grant"
-                    && error_description.contains("refresh token")
-                    && error_description.contains("expired")
-                {
-                    AuthError::OktaRefreshTokenExpired {
-                        message: error_description,
-                    }
-                } else {
-                    AuthError::OpenIDConnectError {
-                        message: "Failed to contact token endpoint".to_string(),
+        .await;
+
+    let token_response = match token_exchange_result {
+        Ok(token_response) => token_response,
+        Err(exchange_error) => {
+            let error = match exchange_error {
+                openidconnect::RequestTokenError::ServerResponse(error) => {
+                    let error_description = error.to_string();
```

Comment:
I think we should also dump the whole error object into debug log. What do you think ?